package.json

```diff
@@ -25,6 +25,9 @@
   "scripts": {
     "commit": "git-cz",
     "version": "conventional-changelog --preset angular --infile CHANGELOG.md --same-file && git add CHANGELOG.md",
+    "test": "yarn test:local",
+    "test:local": "yarn link && cd tests && ./instantsearch.js.sh && ./react-instantsearch.sh && ./vue-instantsearch.sh",
+    "test:saucelabs": "yarn link && cd tests && (export SAUCELABS=1 && ./instantsearch.js.sh && ./react-instantsearch.sh && ./vue-instantsearch.sh)",
```

Comment:
Would this work?

```suggestion
    "test:saucelabs": "SAUCELABS=1 yarn test:local",
```

Reply:
I was convinced it was not working but after testing it seems to work after all :+1: